charts/alerts/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: alerts-chart
 description: A Helm chart for Kubernetes
 type: application
-version: 0.1.0
+version: 0.1.6
```

Comment:
Definitely not needed for this PR, but I wonder if it makes sense to have this version number in a central location for all charts so it can be updated once and read everywhere.

Reply:
Yes, I think that's a good idea!